src/modules/displayBook.js

```diff
@@ -0,0 +1,17 @@
+const displayBook = () => {
+  const booklist = document.querySelector('.booklist');
+  let bookArray = [];
+  if (localStorage.getItem('bookOfLife')) {
+    bookArray = JSON.parse(localStorage.getItem('bookOfLife'));
+  }
+
+  let content = '';
+  if (bookArray.length > 0) {
+    bookArray.forEach((element) => {
+      content += `<li>${element.title} ${element.author} <span>remove</span></li>`;
```

Comment:
- [ ] As we discussed over the call. The remove button should be functional. It should be able to remove a book from the localStorage and consequently, from the UI. Having it as `<span>remove</span>` is not semantically correct, please make it a `<button>` and then add an event listener to it that should enable it to delete a book.